web-app/pages/api/invalidate-all-certs-for-account.ts

```diff
@@ -0,0 +1,51 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { Certificate } from '../../helpers/certificate';
+import { AccountId, apiKey, gas, getNftContract, HTTP_ERROR, HTTP_SUCCESS, NFT, rejectAsUnauthorized } from '../../helpers/near';
+
+const apiKeyHeaderName = 'x-api-key'; // Although the user interface of Integromat shows the capitalization as "X-API-Key", inspecting the actual header reveals that lowercase is used.
+
+async function invalidateAllCertsForAccount(accountId: AccountId) {
+  const contract = await getNftContract();
+  const tokens = await (contract as NFT).nft_tokens_for_owner({ account_id: accountId });
+
+  try {
+    await Promise.all(
+      tokens.map((token: Certificate) =>
+        (contract as NFT).cert_invalidate(
+          // https://github.com/near/near-api-js/issues/719
+          { token_id: token.token_id }, // `memo` here?
```

Comment:
Do we want to put anything in the `memo` field?

Reply:
@encody Thanks for doing this. I'm not clear on what `memo` is for. I don't know of anything we'd want to put there.

Reply:
It's just a human-readable metadata field, afaik. It could be populated with something like `"Invalidated by Integromat hook"` or something like that.